fiftyone/utils/huggingface.py

```diff
@@ -64,6 +59,33 @@
 logger = logging.getLogger(__name__)
 
 
+def list_hub_datasets(info=False):
+    """Lists all FiftyOne datasets available on the Hugging Face Hub.
+
+    This method includes all datasets that are tagged to the ``FiftyOne``
+    library in Hugging Face.
+
+    Examples::
+
+        from fiftyone.utils.huggingface import list_hub_datasets
+
+        datasets = list_hub_datasets()
+        print(datasets)
+
+    Args:
+        info (False): whether to return dataset names (False) or
+            ``huggingface_hub.hf_api.DatasetInfo`` objects (True)
+
+    Returns:
+        a list of dataset names or objects
+    """
+    datasets = list(hfh.list_datasets(filter="library:fiftyone"))
```

Comment:
This used to be `author="Voxel51"` but I switched it to `filter="library:fiftyone"` so that we include all FiftyOne-compatible datasets